package.json

```diff
@@ -2,9 +2,9 @@
   "name": "healthyminds",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "src/app.js",
   "scripts": {
-    "start": "nodemon app.js"
+    "start": "nodemon src/app.js"
```

Comment:
start script should be the production script, so it shouldnt have nodemon it should use node instead, nodemon is for development